<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="primaryStackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1201</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="loginScreen">
     <widget class="QStackedWidget" name="loginStackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1201</width>
        <height>701</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="start">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>160</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>start</string>
        </property>
       </widget>
       <widget class="QPushButton" name="enterButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>320</y>
          <width>251</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string>enter</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="login">
       <widget class="QLineEdit" name="usernameEntry">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>280</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>190</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>login</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordEntry">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>310</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>280</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>310</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QPushButton" name="loginButton">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>340</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>login</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="loginInputStackedWidget">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>380</y>
          <width>181</width>
          <height>81</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="unTested"/>
        <widget class="QWidget" name="invalideInput">
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>161</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Invalid username or password</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="bbFan">
     <widget class="QStackedWidget" name="bbFanStackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1201</width>
        <height>701</height>
       </rect>
      </property>
      <widget class="QWidget" name="home">
       <widget class="QPushButton" name="displayTeamsButton">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>380</y>
          <width>191</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Display Teams</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>220</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>bbFan</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vacationButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>380</y>
          <width>191</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Vacation</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logoutButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="vacation">
       <widget class="QStackedWidget" name="vacationStackedWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>1201</width>
          <height>691</height>
         </rect>
        </property>
        <widget class="QWidget" name="planning">
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>610</x>
            <y>40</y>
            <width>141</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Edit order of visits</string>
          </property>
         </widget>
         <widget class="QPushButton" name="vacationBackButton">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>30</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
         <widget class="QPushButton" name="takeTripButton">
          <property name="geometry">
           <rect>
            <x>870</x>
            <y>400</y>
            <width>191</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>Take Trip</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>40</y>
            <width>191</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Select Teams to Visit</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>630</x>
            <y>20</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Vacation</string>
          </property>
         </widget>
         <widget class="QPushButton" name="optimizeTripButton">
          <property name="geometry">
           <rect>
            <x>870</x>
            <y>290</y>
            <width>191</width>
            <height>71</height>
           </rect>
          </property>
          <property name="text">
           <string>Optimize Trip</string>
          </property>
         </widget>
         <widget class="QListWidget" name="selectTeamsListWidget">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>80</y>
            <width>311</width>
            <height>531</height>
           </rect>
          </property>
         </widget>
         <widget class="QListWidget" name="editTeamsListWidget">
          <property name="geometry">
           <rect>
            <x>500</x>
            <y>80</y>
            <width>311</width>
            <height>531</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="checkAllButton">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>80</y>
            <width>101</width>
            <height>91</height>
           </rect>
          </property>
          <property name="text">
           <string>Check All</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="taking"/>
        <widget class="QWidget" name="end"/>
       </widget>
      </widget>
      <widget class="QWidget" name="displayTeams">
       <widget class="QPushButton" name="displayTeamsBackButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>10</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Display Teams</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_36">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>460</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
       <widget class="QListWidget" name="souvenirList">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>480</y>
          <width>211</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="priceList">
        <property name="geometry">
         <rect>
          <x>1090</x>
          <y>480</y>
          <width>91</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="leagueBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>560</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="openRoofOnly">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>590</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Open Roof Only</string>
        </property>
       </widget>
       <widget class="QComboBox" name="sortTypeBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>470</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>460</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Souvenir</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="centerFieldStacked">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>450</y>
          <width>231</width>
          <height>231</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_2"/>
        <widget class="QWidget" name="hidden_2">
         <widget class="QListWidget" name="stadiumNameList">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>151</width>
            <height>211</height>
           </rect>
          </property>
         </widget>
         <widget class="QListWidget" name="centerFieldList">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>10</y>
            <width>61</width>
            <height>211</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QTableWidget" name="displayTable">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>1171</width>
          <height>401</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>540</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>League</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>450</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Sort Type</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="adminHome">
       <widget class="QPushButton" name="adminVacationButton">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>370</y>
          <width>191</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Vacation</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>210</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>admin</string>
        </property>
       </widget>
       <widget class="QPushButton" name="manageTeamsButton">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>370</y>
          <width>191</width>
          <height>91</height>
         </rect>
        </property>
        <property name="text">
         <string>Manage Teams</string>
        </property>
       </widget>
       <widget class="QPushButton" name="adminLogoutButton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="manageTeams">
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>10</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>manage Teams</string>
        </property>
       </widget>
       <widget class="QPushButton" name="manageTeamsBackButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addTeamsButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>460</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Teams</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteTeamButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>520</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete Team</string>
        </property>
       </widget>
       <widget class="QListWidget" name="priceList_2">
        <property name="geometry">
         <rect>
          <x>1000</x>
          <y>480</y>
          <width>91</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addItemButton">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>480</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Item</string>
        </property>
       </widget>
       <widget class="QListWidget" name="souvenirList_2">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>480</y>
          <width>211</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>870</x>
          <y>460</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Souvenir</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_33">
        <property name="geometry">
         <rect>
          <x>1030</x>
          <y>460</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Price</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="addItemWidget">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>520</y>
          <width>171</width>
          <height>91</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="hide_2"/>
        <widget class="QWidget" name="addTime_2">
         <widget class="QLabel" name="label_34">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Price</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_35">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>47</width>
            <height>13</height>
           </rect>
          </property>
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="itemNameEntry">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>10</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="itemPriceEntry">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>30</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QDialogButtonBox" name="addItemConfirmationBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>156</width>
            <height>23</height>
           </rect>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QPushButton" name="deleteItemButton">
        <property name="geometry">
         <rect>
          <x>660</x>
          <y>610</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete Item</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="manageTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>1171</width>
          <height>401</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="sortTypeBox_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>480</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>460</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Sort Type</string>
        </property>
       </widget>
       <widget class="QComboBox" name="leagueBox_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>570</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>550</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>League</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="openRoofOnly_2">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>600</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Open Roof Only</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="centerFieldStacked_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>460</y>
          <width>231</width>
          <height>231</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page"/>
        <widget class="QWidget" name="hidden">
         <widget class="QListWidget" name="stadiumNameList_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>151</width>
            <height>211</height>
           </rect>
          </property>
         </widget>
         <widget class="QListWidget" name="centerFieldList_2">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>10</y>
            <width>61</width>
            <height>211</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
